config/variables.ts

```diff
@@ -1,11 +1,18 @@
-export const ESCROW_TOKEN = process.env.ESCROW_TOKEN;
-export const NEXT_PUBLIC_GALOY_URL = process.env.NEXT_PUBLIC_GALOY_URL;
-export const NEXT_PUBLIC_LOCAL_URL = process.env.NEXT_PUBLIC_LOCAL_URL;
-export const NEXT_PUBLIC_ESCROW_WALLET_BTC = process.env.NEXT_PUBLIC_ESCROW_WALLET_BTC;
-export const NEXT_PUBLIC_ESCROW_WALLET_USD = process.env.NEXT_PUBLIC_ESCROW_WALLET_USD;
-export const PGHOST = process.env.PGHOST;
-export const PGDATABASE = process.env.PGDATABASE;
-export const PGUSER = process.env.PGUSER;
-export const PGPASSWORD = process.env.PGPASSWORD;
-export const ENDPOINT_ID = process.env.ENDPOINT_ID;
+export const ESCROW_TOKEN = process.env.ESCROW_TOKEN || "";
```

Comment:
I'm introducing https://github.com/t3-oss/t3-env in the backend for better management/validation of env variable. we could do that here also (in a follow up PR)